MediaPicker/src/main/java/org/wordpress/android/util/UiString.kt

```diff
@@ -0,0 +1,13 @@
+package org.wordpress.android.util
+
+import androidx.annotation.StringRes
+
+/**
+ * [UiString] is a utility sealed class that represents a string to be used in the UI. It allows a string to be
+ * represented as both string resource and text.
+ */
+sealed class UiString {
```

Comment:
Related issue: [Decide if we want to use UiString (#31)](https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/issues/31)